src/encoded/tests/fixtures/shared_fixtures.py

```diff
@@ -0,0 +1,125 @@
+import pytest
+
+####
+# This file is for fixtures that have no specific schema and can be shared across multiples test
+####
+
+@pytest.fixture
+def micro_rna_experiment(
+    testapp,
```

Comment:
why not in the experiment fixtures?

Reply:
I did not put it in experiment because it did not seem to fit just it. For example, it made reference to analysis_step_run_bam and biosample. Do you still have an object?

Reply:
I did not put it in experiment because it references other schemas like biosample and others. Do you still have an objection?

Reply:
Done.